server/repositories/cmsQueries/__tests__/secondaryTagHeaderPageQuery.spec.js

```diff
@@ -0,0 +1,39 @@
+const {
+  SecondaryTagHeaderPageQuery,
+} = require('../secondaryTagHeaderPageQuery');
+
+describe('Basic page query', () => {
+  const query = new SecondaryTagHeaderPageQuery('https://cms/content/1234');
+  describe('url', () => {
+    it('should create the correct url', async () => {
+      expect(query.url()).toStrictEqual(
+        'https://cms/content/1234?include=field_featured_image&fields%5Btaxonomy_term--tags%5D=name%2Cdescription%2Cdrupal_internal__tid%2Cfield_featured_image',
+      );
+    });
+  });
+
+  describe('transform', () => {
+    it('should create correct basic page structure', async () => {
+      const basicPage = {
```

Comment:
meh.. presumably this isn't a basic page?

Reply:
🤠